inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -381,11 +383,15 @@ def download_documents(obj, eng):
     for document in documents:
         filename = document['key']
         url = document['url']
-        downloaded = download_file_to_workflow(
-            workflow=obj,
-            name=filename,
-            url=url,
-        )
+        scheme = urlparse(url).scheme
+        if scheme == 'file':
+            downloaded = copy_file_to_workflow(obj, filename, url)
```

Comment:
do we really need two different methods here? cannot you replace `requests` by `fsopen` in `download_file_to_workflow`?

Reply:
I didn't want to touch any existing functions, the file handling should be refactor asap

Reply:
I was thinking the same too, that we don't need both anymore with fsopen. Not 100% sure if it's completely compatible though.

Reply:
we have already logic for handling errors
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/workflows/utils/__init__.py#L210-L230

IMO modifying that might have other side effects which we don't know, we don't reliable tests at the moment